src/search/FindInFiles.js

```diff
@@ -93,7 +93,19 @@ define(function (require, exports, module) {
     }
 
     function nodeFileCacheComplete(event, numFiles, cacheSize) {
-        var projectName = ProjectManager.getProjectRoot().name || "noName00";
+        if (/\/test\/SpecRunner\.html$/.test(window.location.pathname)) {
+            // Ignore the event in the SpecRunner window
+            return;
+        }
+
+        var projectRoot = ProjectManager.getProjectRoot(),
+            projectName = projectRoot ? projectRoot.name : null;
```

Comment:
If we don't have the project root, projectName is null and thus we get the error message below.